account-kit/infra/src/gas-manager.ts

```diff
@@ -38,6 +38,8 @@ export const AlchemyPaymasterAddressV1 =
  *
  * @param {Chain} chain The chain for which the paymaster address is required
  * @returns {Address} The Alchemy paymaster address corresponding to the specified chain
+ *
+ * @deprecated This chain list in this function is no longer maintained since the ERC-7677 middleware is typically used to resolve the paymaster address
```

Comment:
is this what's used in the new (old) middleware too?

Reply:
the old version of the middleware that used `alchemy_requestGasAndPaymasterAndData` was using this for building the dummy paymaster data.. i discovered it's an issue when the chains team was testing this in ws-tools, b/c it would fail for any other chain.

in order to prevent having to maintain this, the new version of the `alchemy_requestGasAndPaymasterAndData` middleware will fall back to using the 7677 dummyPaymasterAndData middleware (calling `pm_getPaymasterStubData`) if there is a gasEstimator or feeEstimator override, otherwise it just does a no-op since it's not actually needed.